code/game/machinery/doors/brigdoors.dm

```diff
@@ -263,15 +263,12 @@
 
 	return .
 
-/obj/machinery/door_timer/attackby(obj/item/O as obj, user as mob)
+/obj/machinery/door_timer/attackby(obj/item/O as obj, var/mob/user as mob)
 	if( istype( O, /obj/item/weapon/paper/incident ))
 		if( !incident )
 			if( import( O, user ))
-				usr.drop_item()
-				O.loc = src
-
 				ping( "\The [src] pings, \"Successfully imported incident report!\"" )
-				qdel( O )
+				user.drop_from_inventory(O,get_turf(src),TRUE)

```

Comment:
Drop from inventory != qdel wtf bbq broskidoodle.

Reply:
Oh, I neglected to mention that drop_from_inventory also now has a (qdel) proc built in if the third argument is set to TRUE.

Reply:
...why not just qdel() the object and let Destroy() call DFI.

Reply:
It's faster to check a bool than it is to check whether or not the object is being held by a person and to call a drop_from_inventory.

Reply:
This isn't performance critical code.